react-components/toolbar-filter/step3.md

```diff
@@ -1,131 +1,106 @@
-The individual components placed inside of a toolbar are each wrapped in a `DataToolbarItem` component. `DataToolbarItems` can be passed an optional `variant` prop to format specific type of component it wraps such as a search filter, a pagination component, a bulk select, or an overflow menu. By wrapping components in the `DataToolbarItem` component and adding the proper variant, you are ensuring the components are spaced appropriately.
+Toolbar groups are used to group sets of like items to create desired associations and to enable items to respond together to changes in viewport width. A `DataToolbarGroup` is passed an optional prop `variant` to format particular types of groups of elements such as filter groups, button groups, or icon button groups.
 
-## Task
+This step will group `DataToolbarItem` components into `DataToolbarGroup` components and apply appropriate `variant` props.
 
-We will use the state of our ToolbarDemoApp to manage some of the items in our `DataToolbar`.
+1) **Locate the following two `DataToolbarItems` containing `CheckBoxSelect` components:**
 
-1) Locate the code that looks like the following:
 <pre class="file">
-  constructor(props) {
-    super(props);
-  }
+&lt;DataToolbarItem&gt;
+  &lt;CheckboxSelect
+    onSelect={this.onSelect}
+    type=&quot;Status&quot;
+    selections={filters.status}
+    options={statusOptions}
+  /&gt;
+&lt;/DataToolbarItem&gt;
+&lt;DataToolbarItem&gt;
+  &lt;CheckboxSelect
+    onSelect={this.onSelect}
+    type=&quot;Risk&quot;
+    selections={filters.risk}
+    options={riskOptions}
+  /&gt;
+&lt;/DataToolbarItem&gt;
 </pre>
 
-2) Replace ToolbarDemoApp constructor with the following snippet.
-
-<pre class="file" data-target="clipboard">
-  constructor(props) {
-    super(props);
-    this.state = {
-      filters: {
-        name: [],
-        risk: [&quot;Low&quot;],
-        status: [&quot;New&quot;, &quot;Pending&quot;]
-      }
-    };
-  }
-</pre>
+2) **Wrap both the `DataToolbarItem` components containing the `CheckboxSelect` filters in the same `DataToolbarGroup` component**
```

Comment:
Its a little confusing as to whether to wrap both components in DataToolbarGroup or use one wrapper around both.

```suggestion
2) **Wrap both the `DataToolbarItem` components containing the `CheckboxSelect` filters in one `DataToolbarGroup` component**
```